### PR TITLE
WS2-1237: Add show borders field for card paragraphs.

### PR DESCRIPTION
**Pull request by juanmitriatti**

### CODE REVIEW

**Comment by juanmitriatti** [comment]
Ticket: https://asudev.jira.com/browse/WS2-1237

**Comment by mlsamuelson** [comment]
HOLD on merging until after 2.8 hotfix release of WS2

File: config/install/core.entity_view_display.paragraph.card.default.yml [hunk 1]
```diff
@@ -8,21 +8,47 @@ dependencies:
...
+  field_clickable:
+    type: boolean
+    label: above
+    settings:
+      format: default
+      format_custom_false: ''
+      format_custom_true: ''
+    third_party_settings: {  }
+    weight: 0
+    region: content
```
Comment (mlsamuelson) [inline config/install/core.entity_view_display.paragraph.card.default.yml:45]: @juanmitriatti Is this sneaking back in after it was removed in the [WS2-1529 PR](https://github.com/ASUWebPlatforms/webspark-module-webspark_blocks/commit/c45b8c94a300551f7457867daf6e43d9af521533#diff-7cce76195417c597939dd9a4465a2ce25e7d3cfd248952c0a29a6c4302d22669L24-L46) ?

I see other instances of the `field_card_link` field being added in these yaml's and I'm concerned it's accidental. Can you please check?

**Review by mlsamuelson — CHANGES_REQUESTED** [review]

File: config/install/core.entity_view_display.paragraph.card.default.yml (hunk 1 above)
Comment (juanmitriatti) [reply config/install/core.entity_view_display.paragraph.card.default.yml:45]: Hi @mlsamuelson 
Let me review please.
Thanks

Comment (juanmitriatti) [reply config/install/core.entity_view_display.paragraph.card.default.yml]: Hi @mlsamuelson 
I just pushed a commit reverting the field clickable and field card link from this PR.
Thanks

**Review by mlsamuelson — APPROVED** [review]

**Pull request merged**